jbpm-wb-forms/jbpm-wb-forms-backend/src/main/java/org/jbpm/workbench/forms/display/backend/FormRendererProxyServlet.java

```diff
@@ -107,8 +112,28 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws S
             String processId = extractValue(processIdPattern, pathInfo);
             ProcessServicesClient processServicesClient = getClient(serverTemplateId,                                                      
                                                                     ProcessServicesClient.class);
-            
-            Long responseBody = processServicesClient.startProcess(containerId, processId, new StringContentMap(body));
+            Long responseBody = 0L;
```

Comment:
Would you mind add a test the Servlet please?

Reply:
Done